bylaws.tex

```diff
@@ -301,10 +301,9 @@
 If the member is still unsatisfied after being heard by the Executive Board, the appeal may be brought to the attention of the ResLife Advisor.
 
 \bsection{Expectations of House Members}
-Each member is required to pay House dues as stated in \ref{Collection of House Dues}, attend all House Meetings, and attend at least twenty-five (25) of the permanent director meetings.
-Members are also required to actively participate in a majority of House activities.
+Each member is required to pay House dues as stated in \ref{Collection of House Dues}, attend all House Meetings, and attend at least twenty-five (25) meetings consiting of the seven permanent director meetings and open Executive Board as defined in \ref{Open Executive Board}.
 \\* \\*
-The member must participate significantly in at least one major House project during the academic year.
```

Comment:
I feel like this change alone could be made as non-semantic, considering this wording doesn't appear to show itself anywhere else in the document and is inconsistent with the following line. If the rest of these changes are shat upon I'll probably just make a PR for that.

Reply:
If you could separate out your non-semantic changes into a separate PR, we could at least get those changes merged in while we wait to vote on the rest.